nav2_costmap_2d/plugins/costmap_filters/costmap_filter.cpp

```diff
@@ -63,15 +63,25 @@ void CostmapFilter::onInitialize()
   try {
     // Declare common for all costmap filters parameters
     declareParameter("enabled", rclcpp::ParameterValue(true));
+    declareParameter("enable_service", rclcpp::ParameterValue("enable"));
     declareParameter("filter_info_topic", rclcpp::PARAMETER_STRING);
     declareParameter("transform_tolerance", rclcpp::ParameterValue(0.1));
 
     // Get parameters
     node->get_parameter(name_ + "." + "enabled", enabled_);
     filter_info_topic_ = node->get_parameter(name_ + "." + "filter_info_topic").as_string();
+    std::string enable_service_name;
+    node->get_parameter(name_ + "." + "enable_service", enable_service_name);
```

Comment:
Don't parameterize the name if its going to be put under the namespace of the costmap filter anyway. That way these are always standardized service names that people can rely on. 

How about `toggle_filter`?

Reply:
Agree with this name. Made an update in https://github.com/ros-planning/navigation2/pull/3229/commits/33aeff9ebe1cd4c3e46a0ac2cbb81b3b07eb4db4